man/jail.conf.5

```diff
@@ -146,6 +146,12 @@ Ensure syslog or the program that generates the log file isn't configured to com
 .B logencoding
 encoding of log files used for decoding. Default value of "auto" uses current system locale.
 .TP
+.B banaction
+default banning action (iptables-multiport) for all jails specified in the \fI[DEFAULT]\fR section.
```

Comment:
> Now it is misleading

Now it is available (before it was at all not documented) ... but I've corrected my "mistake" in 576719198805063c1c3b30a7cb31f7beedd7c2f8.